### PR TITLE
Re-do first paragraph of 'contributing packages'.

### PR DESCRIPTION
**Pull request by chicken-biryani**
PR Checklist:

- [X] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [X] put any other relevant information below

The first paragraph of ['contributing packages'](https://conda-forge.org/docs/maintainer/adding_pkgs.html) seemed to require clarity. The PR tries to give a clearer picture by giving the title 'Process' to the paragraph and further divides the already existing paragraph into steps so that a person submitting a package knows where they stand.

The outcome:

Screenshot (50)

PS: I haven't created an issue for the PR and it would be really great if someone could suggest how to make it look better.

### CODE REVIEW

**Comment by chicken-biryani** [comment]
Hi @beckermr @kathatherine and @BastianZim hope this PR is helpful. Would appreciate your feedback.

File: src/maintainer/adding_pkgs.rst [hunk 1]
```diff
@@ -3,15 +3,19 @@
 Contributing packages
 *********************
 
+Process
+===================
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:7]: You should make the length of any header underlines be the same length as the text. Not a horrible thing, but it does cause warnings in the build log that can take up a lot of space and make it difficult to spot other problems.

File: src/maintainer/adding_pkgs.rst [hunk 2]
```diff
@@ -3,15 +3,19 @@
...
 
-To submit a package to the ``conda-forge`` channel, add its ``recipe`` and licence to the ``staged-recipes`` repository and create a pull request. Once the pull request is merged, the package becomes available
-on the ``conda-forge`` channel. Note that contributing a package makes you the ``maintainer`` of that package.
+To submit a package to conda-forge one has to follow the steps:
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:9]: "To submit a package to conda-forge, follow these steps:"

It's better to use what we call second person imperative in documentation. Basically, write like you're giving an order (but in a nice way, of course). :)

File: src/maintainer/adding_pkgs.rst [hunk 3]
```diff
@@ -3,15 +3,19 @@
...
 
-A maintainer is responsible for maintaining the feedstock repository and packages as well as their future versions and has push access to the feedstock repositories of only the packages it maintains. 
-You can learn more about the roles of a maintainer `here. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__
+* Step 1. The staging process (add recipe and license)
+ To submit a package to the ``conda-forge`` channel, one has to add its recipe and license, with the help of the `staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_, to the `staged-recipes repository <https://github.com/conda-forge/staged-recipes>`_ and create a pull request.
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:12]: Same second person imperative tense here as above. "To submit a package to the conda-forge channel, add its recipe and license..."

File: src/maintainer/adding_pkgs.rst [hunk 4]
```diff
@@ -3,15 +3,19 @@
...
 
-The sections below provide detailed instructions on contributing packages to conda-forge.
+* Step 2. Post Staging process.
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:14]: Leave "staging" lowercase.

File: src/maintainer/adding_pkgs.rst [hunk 5]
```diff
@@ -3,15 +3,19 @@
...
+Process
+===================
 
-To submit a package to the ``conda-forge`` channel, add its ``recipe`` and licence to the ``staged-recipes`` repository and create a pull request. Once the pull request is merged, the package becomes available
-on the ``conda-forge`` channel. Note that contributing a package makes you the ``maintainer`` of that package.
+To submit a package to conda-forge one has to follow the steps:
 
-A maintainer is responsible for maintaining the feedstock repository and packages as well as their future versions and has push access to the feedstock repositories of only the packages it maintains. 
-You can learn more about the roles of a maintainer `here. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__
+* Step 1. The staging process (add recipe and license)
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:11]: If the other two steps end in a period, have one here, too, to be consistent.

File: src/maintainer/adding_pkgs.rst [hunk 6]
```diff
@@ -3,15 +3,19 @@
...
+ To submit a package to the ``conda-forge`` channel, one has to add its recipe and license, with the help of the `staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_, to the `staged-recipes repository <https://github.com/conda-forge/staged-recipes>`_ and create a pull request.
 
-The sections below provide detailed instructions on contributing packages to conda-forge.
+* Step 2. Post Staging process.
+ To know what happens once a PR has been merged have a look at our `post staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#post-staging-process>`_.
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:15]: Needs a comma between "merged" and "have".

File: src/maintainer/adding_pkgs.rst [hunk 7]
```diff
@@ -3,15 +3,19 @@
...
 
+* Step 3. Maintaining the package.
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:17]: Needs a line between the step title and the rest of the text.

File: src/maintainer/adding_pkgs.rst [hunk 8]
```diff
@@ -3,15 +3,19 @@
...
+ Note that contributing a package to ``conda-forge`` makes you the maintainer of that package. A maintainer is responsible for maintaining the feedstock repository and packages as well as their future versions and has to push access to the feedstock repositories of only the packages it maintains. Learn more about the `roles of a maintainer <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`_.
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:18]: Who does "it" refer to in this paragraph? The maintainer or the feedstock repo?

File: src/maintainer/adding_pkgs.rst [hunk 9]
```diff
@@ -101,7 +105,7 @@ After merging the :term:`PR`, our :term:`CI` infrastructure will build the packa
 
 
 Post staging process
---------------------
+===================
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:108]: Make sure that header underline is the right length.

File: src/maintainer/adding_pkgs.rst [hunk 10]
```diff
@@ -110,7 +114,7 @@ Post staging process
 
 
 Maintainer role
----------------
+===================
```
Comment (kathatherine) [inline src/maintainer/adding_pkgs.rst:117]: And here, too.

**Review by kathatherine — CHANGES_REQUESTED** [review]
I think this will be very helpful. Let me know if you have questions!

**Comment by chicken-biryani** [comment]
@kathatherine I have made the changes. A few pointers, I felt adding about the roles of the maintainer in the Process as well as in the Maintainer role was redundant. What do you think?

**Review by kathatherine — APPROVED** [review]
Yes, I agree that was redundant. Thanks so much!

**Pull request merged**